scripts/generate-component-config/generate-component-config.js

```diff
@@ -35,24 +36,29 @@ commander
  *     dir/packages/ * /examples/ * /test-examples
  */
 const compiledDirPattern = `{examples,${path.join('examples', '*')}}`;
+const terraReposAsPackages = path.join('node_modules', '{terra-core,terra-clinical,terra-framework,terra-consumer}');
```

Comment:
Ideally, we'd remove any reference to our mono repos from this repo and have anything like this set by configuration sent in by the repo consuming terra-site.

Reply:
I have no strong option towards this change. The thought was terra-ui and all theming repos would require the same regex pattern, which long when listing each repo, especially when we add patterns to remove any '|' for windows users.

Reply:
It feels weird to include it here, but the use would be in a few places.